### ISSUE TITLE
Autopilot wobbles up and down after reaching Altitude Hold

### ISSUE THREAD

**Issue by morphelianus** [MuMech/MechJeb2#1342]
## MechJeb Version (DEV RELEASE)
2.10.0.0-1001

## KSP Version
Stock 1.10.1.2939 de-de
Making History 1.10.1
Breaking Ground 1.5.1

## Description
After engaging Aircraft Autopilot and reaching the set Altitude Hold of 7000 meters the airplanes starts flying a parable instead of keeping the set target altitiude. Like a bullwipe it swinged up and could even increase to an amplitude of 300 to 500 meters. I was only able to avoid it by decreasing Vertical Speed to 1 m/s and then to 0 m/s.
https://youtu.be/ghMSoK4600s

After a couple of minutes flying at a stable altitude of 7000 meters I set Vertical Speed back to 10 m/s and increased the time warp which caused again a parable flight.
https://youtu.be/riv0F3MhrPw

In some cases the airplane stops climbing at 6000 meters while vertical speed turns into negative values. After a drop of 500 meters it starts climbing to 7000 meters and above and remains in a parable flight. All parameters of the airplane, like speed or thurst are fine. Hence I can't explain.

This also causes a significant waste of fuel as the airplanes goes down and up and down and up.

## Replication Case
I took KSC runway 27 and engaged Aircraft Autopilot right after lift off heading 180° and then turning to 265°, altidude hold set to 7000 meters, V/S to 20 m/s, bank angle 30° and horizontal speed hold set to 120 m/s. The airplane should be capable of all of these settings. After reaching 6500 meters I set V/S back to 10 m/s. After reaching 7000 meters the above described behavior occured. I did not change any other settings. Just played a little bit with V/S in order to try stopping the annoying parable flight.

## KSP.log
[KSP.log](https://github.com/MuMech/MechJeb2/files/5010286/KSP.log)

**Comment by morphelianus** [comment]
#1163 seems to be the same issue

**Comment by morphelianus** [comment]
I did some more tests and it seems that the issue depends somehow on the airplane configuration or layout. While the error occurs whenever I fly my K-6 Pigeon I was unable to reproduce the issue with my K-5 Hawk. I added both airplanes. May it helps to identify the issue and provide a solution.
[K-4 Hawk MJ.craft.txt](https://github.com/MuMech/MechJeb2/files/5011336/K-4.Hawk.MJ.craft.txt)
[K-5 Pigeon.craft.txt](https://github.com/MuMech/MechJeb2/files/5011337/K-5.Pigeon.craft.txt)
